src/Altinn.AccessManagement.Core/Services/Altinn2RightsService.cs

```diff
@@ -112,10 +113,23 @@ private async Task<List<RightDelegation>> MapDelegationResponse(IEnumerable<Dele
             var resourcePath = delegation.ResourceId.Split("/");
             if (delegation.ResourceType.Contains("AltinnApp", StringComparison.InvariantCultureIgnoreCase) && resourcePath.Length > 1)
             {
-                entry.Resource.AddRange(resources
+                var delegatedResource = resources
                     .Where(a => a.AuthorizationReference.Exists(p => p.Id == AltinnXacmlConstants.MatchAttributeIdentifiers.OrgAttribute && p.Value == resourcePath[0]))
-                    .First(a => a.AuthorizationReference.Exists(p => p.Id == AltinnXacmlConstants.MatchAttributeIdentifiers.AppAttribute && p.Value == resourcePath[1]))
-                    .AuthorizationReference);
+                    .Where(a => a.AuthorizationReference.Exists(p => p.Id == AltinnXacmlConstants.MatchAttributeIdentifiers.AppAttribute && p.Value == resourcePath[1]));
```

Comment:
I stedet for å splitte og sjekke Org og AppName mot AuthorizationReference, kan man sjekke hele org/appname mot 
ResourceReferences av typen ApplicationId på ressursen.

eksempel resourceReferences:
```
"resourceReferences": [
      {
        "referenceSource": "Altinn3",
        "reference": "ttd/endring-av-navn-v2",
        "referenceType": "ApplicationId"
      }
    ]
```

Reply:
Hvis vi i det hele tatt skal slå opp via ressursen... Alternativt kan vi jo bare alltid ha gjøre splitt og hardkode resource som i else under. Da hadde vi heller aldri fått problemet (før/om vi noen gang kommer dit at vi skal endre fra id/value til type/value)

Reply:
stemmer, jeg kan legge til objektet i altinn appen > resourceReferences så ligger det testdataen. Ja, hva bør vi gå for? parse id og returnere?